front_end/src/app/(campaigns-registration)/bridgewater/components/registration-status.tsx

```diff
@@ -0,0 +1,255 @@
+      >
+        {isEligible ? "Eligible *" : "Not Eligible *"}
+      </span>
+    </div>
+  );
+};
+
+const LoggedInAndRegisteredFragment: FC<{ eligibleBoth: boolean }> = ({
+  eligibleBoth,
+}) => {
+  const { setCurrentModal } = useModal();
+
+  return (
+    <>
+      <HeadingText
+        heading={"You’re all set!"}
+        subheading={
+          "Your registration is confirmed. Here’s your leaderboard eligibility:"
+        }
+      />
+
+      <div className="flex w-full flex-col gap-1 text-base text-gray-800 dark:text-gray-800-dark">
+        <EligibilityBox isEligible={true}>Open Prize Pool</EligibilityBox>
+
+        <EligibilityBox isEligible={eligibleBoth}>
+          Undergrad Prize Pool
+        </EligibilityBox>
+      </div>
+
+      <p className="mb-0 mt-5 text-sm text-gray-0 dark:text-gray-0-dark xs:text-base sm:text-sm md:text-lg ">
+        Ready to get started? Try the{" "}
+        <span
+          className="cursor-pointer underline"
+          onClick={() => setCurrentModal({ type: "onboarding" })}
+        >
+          Forecasting Tutorial
+        </span>{" "}
+        or explore some{" "}
+        <Link href="/tournaments/bridgewater/">Warmup Questions</Link> to
+        sharpen your skills!
+      </p>
+
+      <div className="flex items-start gap-1.5 text-xs text-blue-400 dark:text-blue-400-dark">
+        <span className="my-0">*</span>
+        <p className="my-0 text-left">
+          Displayed prize-pool eligibility is based solely on your undergraduate
+          status. For full eligibility details see the{" "}
+          <Link href="contest-rules">contest rules</Link>.
+        </p>
+      </div>
+    </>
+  );
+};
+
+interface RegisterAndStatusProps {
+  className?: string;
+  currentUser: CurrentUser | null;
+}
+
+export const RegisterAndStatus: FC<RegisterAndStatusProps> = ({
+  className,
+  currentUser,
+}) => {
+  const [registerDialogVisible, setRegisterDialogVisible] = useState(false);
+  const [signupDialogVisible, setSignupDialogVisible] = useState(false);
+
+  const router = useRouter();
+  const [signupCompleteEmail, setSignupCompleteEmail] = useState<string | null>(
+    null
+  );
+  const campaigns = currentUser?.registered_campaigns.filter(
+    ({ key }) => key == CAMPAIGN_KEY
+  );
+
+  const registered = campaigns && campaigns.length > 0;
+  const campaignDetails =
+    registered && (campaigns[0]?.details as { undergrad: boolean });
+
+  const SignupCompleteFragment: FC<{ email: string }> = ({ email }) => (
+    <div>
+      <h2 className="my-0 text-base text-gray-0 dark:text-gray-0-dark xs:text-lg sm:text-lg md:text-2xl  xl:text-3xl">
+        Check your inbox!
+      </h2>
+      <p className="mb-0 mt-5 text-sm text-gray-0 dark:text-gray-0-dark xs:text-base sm:text-sm md:text-lg ">
+        To complete your registration, confirm your email address. We’ve sent a
+        confirmation link to your inbox{" "}
+        <span className="italic">(john.${email})</span>. Once confirmed, you’ll
```

Comment:
Double-check, if the text here correct? Shouldn't it be `({email})`? With current implementation the app will output: `(john.$john.doe@gmail.com)`

Reply:
Good point, just noticed this one and already fixed. Pushing a new change. Txh!